tss-esapi/src/utils/mod.rs

```diff
@@ -23,6 +23,11 @@ use crate::{Context, Error, Result, WrapperErrorKind};
 use std::convert::TryFrom;
 use zeroize::Zeroize;
 
+mod credential;
+pub mod kdf;
```

Comment:
Since these make use of the Rust Crypto dependencies, would it make sense to gate them on that feature too?